packages/runtime-core/src/scheduler.ts

```diff
@@ -147,6 +148,9 @@ export function flushPreFlushCbs(
       if (__DEV__ && checkRecursiveUpdates(seen!, cb)) {
         continue
       }
+      if (instance && cb.id !== instance.uid) {
+        continue
+      }
```

Comment:
Shouldn't this come before the `checkRecursiveUpdates` check? We shouldn't be incrementing the count in `seen` unless we're actually running the job.

Maybe something like:

```ts
if (cb && cb.pre && (!instance || cb.id === instance.id)) {
```

Reply:
Thanks. Updated. 
I prefer to this condition `instance && cb.id !== instance.uid` that more clearer.